src/main/java/uk/gov/ida/verifyserviceprovider/services/ClassifyingAssertionTranslator.java

```diff
@@ -9,38 +9,17 @@
 public class ClassifyingAssertionTranslator implements AssertionTranslator {
 
     private final VerifyAssertionTranslator verifyAssertionTranslator;
-    private final EidasAssertionTranslator eidasAssertionTranslator;
-    private final EidasUnsignedAssertionTranslator eidasUnsignedAssertionTranslator;
 
 
     public ClassifyingAssertionTranslator(
```

Comment:
This class just seems to be a wrapper around the VerifyAssertionTranslator class. Could we cut out the middleman and drop this class completely?

Reply:
Yep, removed this class with no affect